app/models/country.rb

```diff
@@ -7,4 +7,18 @@ def api_json
       description:,
     }.as_json
   end
+
+  def self.populate
+    CountriesPopulator.call
+    refresh_text_searchable
+  end
+
+  def self.refresh_text_searchable
+    update_query = <<~UPDATESQL.freeze
+      UPDATE countries
+      SET searchable =
+        setweight(to_tsvector(coalesce(description, '')), 'A');
```

Comment:
nit: as we are only searching "description", is `setweight` still required OR is this the recommended pattern?

Reply:
I believe that `char` is the default... On the org search, it's created as
```ruby
to_tsvector('simple', coalesce(name, '')), 'A')
```

But the schema shows
```ruby
to_tsvector('simple'::regconfig, (COALESCE(name, ''::character varying))::text), 'A'::\"char\")
```
Which implies, to me, that it gets added internally unless you specify something else.

As to the 'A', all the other vectors have one so I carried on the tradition! 😄